backend/src/ledgers/data.ts

```diff
@@ -64,14 +66,27 @@ export type LedgerRecord = {
   operation_count: number;
 };
 
-export async function updateLedgers() {
+export async function updateLedgers(isTestnet: boolean) {
+  const horizonServer = getHorizonServer(isTestnet);
+  console.info("Updating ledgers using horizon server URL: ", horizonServer);
+
   for (const interval of intervalTypes) {
-    const cachedData =
-      (await redisClient.get(REDIS_LEDGER_KEYS[interval])) || "[]";
-    const cachedLedgers: LedgerStat[] = JSON.parse(cachedData);
-    let pagingToken = "";
+    const REDIS_LEDGER_KEY = getServerNamespace(
+      REDIS_LEDGER_KEYS[interval],
+      isTestnet,
+    );
+
+    const REDIS_PAGING_TOKEN_KEY_VALUE = getServerNamespace(
+      REDIS_PAGING_TOKEN_KEY[interval],
+      isTestnet,
+    );
 
-    if (cachedLedgers.length < LEDGER_ITEM_LIMIT[interval]) {
+    const cachedData = (await redisClient.get(REDIS_LEDGER_KEY)) || "[]";
+    const cachedLedgers: LedgerStat[] = JSON.parse(cachedData);
+    let pagingToken = isTestnet
```

Comment:
you can leave this as `let pagingToken = ""`, you're setting it to CURSOR_NOW for testing in the else block on line 104